### PR TITLE
add option to drop views during DropCommand

### LINKED ISSUE #602: Handling for Views in db:dump / db:drop
## Feature description

Currently we do not handle views during db:dump operations.
If you run `db:dump --drop-tables` or `db:drop --tables`all existing tables will be removed before the command logic is executed.
If there are views, the views remain. That's expected.

The feature could add additional `--drop-views` and `--views` option to the commands.

## Problem and motivation

Resolves question in #579

### PR DESCRIPTION
**Pull request by torhoehn**
Magerun pull-request check-list:

- [x] Pull request against develop branch (if not, just close and create a new one against it)
- [ ] README.md reflects changes (if any)
- [ ] phar fuctional test (in tests/phar-test.sh)

Summary: (some words as summary)

Fixes #602.

Changes proposed in this pull request:

- Extend the database helper to return views
- Adopt table drop logic also for views in db:drop command

### CODE REVIEW

File: src/N98/Util/Console/Helper/DatabaseHelper.php [hunk 1]
```diff
@@ -610,6 +610,60 @@ public function getTables($withoutPrefix = null)
...
+     * Get list of database tables
+     *
+     * @param bool|null $withoutPrefix [optional] remove prefix from the returned table names. prefix is obtained from
+     *                                 magento database configuration. defaults to false.
+     *
+     * @return array
+     * @throws RuntimeException
+     * @throws \Magento\Framework\Exception\FileSystemException
+     */
+    public function getViews($withoutPrefix = null)
```
Comment (torhoehn) [inline src/N98/Util/Console/Helper/DatabaseHelper.php:623]: just copied from getTables function, has to be tested and developed

**Comment by cmuench** [comment]
@torhoehn nice. I will review that after the next release if it's ok. Or we discuss it in Cologne :-)

**Comment by torhoehn** [comment]
@cmuench I'm sure you could teach me something there. :)

**Comment by cmuench** [comment]
@torhoehn I didn't forget your PR. I plan to merge this feature in the v7.0.0 release.

**Comment by cmuench** [comment]
@torhoehn I remove the "-v" shortcut. This produces an error, because the "-v" option is already reserved as global shortcut of the verbose option in Symfony Commands.
The I rebased the branch with latest develop branch of n98-magerun2 which got a bigger refactoring. Jobs are running. I will tests the PR in the next days.

**Comment by cmuench** [comment]
I tested the drop function.
It's currently not working. I got this error `SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual th  
  at corresponds to your MariaDB server version for the right syntax to use near 'LIKE 'view%'' at line 1`

The generated SQL query looks like this:
`SELECT table_name FROM information_schema.views WHERE table_schema = database() AND LIKE 'view%';`

**Comment by cmuench** [comment]
I close this MR until we have refactored the command.

**Pull request closed without merging**